docs/source/get_involved/governance.rst

```diff
@@ -320,6 +320,45 @@ For more details about our meetings, please go to our `community-council reposit
 
 To contact the CC directly, please send an email to info@sktime.org.
 
+.. _cc-observers:
+
+CC observers
+~~~~~~~~~~
+
+CC (community council) observers are core developers with additional rights and
+responsibilities. Current CC observers are listed in the `community-council
+observers <https://www.sktime.org/en/stable/about/team.html>`__ .
+
+.. _rights-and-responsibilities-4:
+
+Rights and responsibilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Observers have a full view of CC and participate in the decision making process
```

Comment:
I would be specific and clearer. How about (based on your paragraph):

CC observers have a full view of reserved CC proceedings, the private CC channels and the sktime email account.
CC observers can participate in discussions on private CC channels to ensure that more members of the community have direct input on CC decisions.
CC observers do not possess the voting or decision making rights of full CC members.